### ISSUE TITLE
In model.py line 151， got an error: tuple has no attribute "logits"

### ISSUE THREAD

**Issue by Du-danger** [OpenBGBenchmark/OpenBG-CSK#3]
In model.py line 151， got an error: tuple has no attribute "logits".
We fix it and then run normally.
Specially, we replace "logits" with "[0]".
i.e., pred = self.bert(inputs_embeds=embeddings, attention_mask=attention_mask)[0]

I don't know whether the correction is right.
Thank you!

**Comment by qyccc** [comment]
It may caused by different version of transformer lab. From my experience, "logits" is the same with "[0]". You can check the api [here](https://huggingface.co/docs/transformers/model_doc/bert#transformers.BertModel).
